core/src/test/scala/org/apache/spark/scheduler/BarrierTaskContextSuite.scala

```diff
@@ -201,15 +201,25 @@ class BarrierTaskContextSuite extends SparkFunSuite with LocalSparkContext {
     }
   }
 
-  test("barrier task killed") {
+  test("barrier task killed, no interrupt") {
+    val conf = new SparkConf()
```

Comment:
Can this part also go into the helper method to not duplicate it or is it used elsewhere?